native/core/src/parquet/parquet_exec.rs

```diff
@@ -102,10 +104,13 @@ pub(crate) fn init_datasource_exec(
     Ok(Arc::new(DataSourceExec::new(Arc::new(file_scan_config))))
 }
 
-fn get_options(session_timezone: &str) -> (TableParquetOptions, SparkParquetOptions) {
+fn get_options(
+    session_timezone: &str,
+    pushdown_filters: bool,
+) -> (TableParquetOptions, SparkParquetOptions) {
     let mut table_parquet_options = TableParquetOptions::new();
+    table_parquet_options.global.pushdown_filters = pushdown_filters;
     // TODO: Maybe these are configs?
```

Comment:
I think we can remove the TODO, and maybe get rid of the `reorder_filters = true` on the line below. DF still defaults that to false as well so we might not understand the performance implications of it yet. We could add a config for that in a follow up PR and measure the difference.

Reply:
Thanks. I have updated this and also added some notes to the tuning guide.